Kentico/xperience-algolia
# Feat/app settings configuration

### Motivation

Resolves #10 

### Checklist


### How to test

Set `AlgoliaSearchDisableIndexing` app setting to `true` and see no indexing happens.
